package.json

```diff
@@ -17,7 +17,7 @@
         "accessibility-developer-tools": "2.10.0",
         "bluebird": "^3.2.2",
         "jquery": "^2",
-        "a11y-auditor": "pranavjha/a11y-auditor"
+        "a11y-auditor": "latest"
```

Comment:
dont use latest.. `'^0.0.1'` should be fine. `latest` will cause issues if we push a breaking change in `a11y-auditor v1.x.x`